xbmc/GUIInfoManager.cpp

```diff
@@ -6225,38 +6225,38 @@ std::string CGUIInfoManager::GetLabel(int info, int contextWindow, std::string *
     strLabel = GetGameLabel(info);
     break;
   case VIDEOPLAYER_VIDEO_CODEC:
-    if(g_application.m_pPlayer->IsPlaying())
+    if(g_application.GetAppPlayer().IsPlaying())
     {
       strLabel = m_videoInfo.codecName;
     }
     break;
   case VIDEOPLAYER_VIDEO_RESOLUTION:
-    if(g_application.m_pPlayer->IsPlaying())
+    if(g_application.GetAppPlayer().IsPlaying())
     {
       return CStreamDetails::VideoDimsToResolutionDescription(m_videoInfo.width, m_videoInfo.height);
     }
     break;
   case VIDEOPLAYER_AUDIO_CODEC:
-    if(g_application.m_pPlayer->IsPlaying())
+    if(g_application.GetAppPlayer().IsPlaying())
     {
       strLabel = m_audioInfo.codecName;
     }
     break;
   case VIDEOPLAYER_VIDEO_ASPECT:
-    if (g_application.m_pPlayer->IsPlaying())
+    if (g_application.GetAppPlayer().IsPlaying())
     {
       strLabel = CStreamDetails::VideoAspectToAspectDescription(CServiceBroker::GetDataCacheCore().GetVideoDAR());
     }
     break;
   case VIDEOPLAYER_AUDIO_CHANNELS:
-    if(g_application.m_pPlayer->IsPlaying())
+    if(g_application.GetAppPlayer().IsPlaying())
```

Comment:
also not needed. m_audio/videoInfo can be reset to defaults, if player is not playing

Reply:
done